package.json

```diff
@@ -71,9 +71,9 @@
     }
   },
   "dependencies": {
-    "@philipplgh/electron-app-manager": "^0.55.0",
+    "electron-app-manager": "0.57.0",
     "ipfs": "^0.52.2",
     "ipfs-http-gateway": "^0.1.3",
-    "remixd": "^0.2.3-alpha.4"
+    "@remix-project/remixd": "^0.3.0"
```

Comment:
Update remixd version for windows users.

Reply:
updated the version